test/integ/quote-classic.test.ts

```diff
@@ -136,18 +168,25 @@ describe('quote', function () {
     currencyIn: Currency,
     currencyOut: Currency,
     permit?: boolean,
-    chainId = ChainId.MAINNET
+    chainId = ChainId.MAINNET,
+    portion?: Portion
   ): Promise<{
     tokenInAfter: CurrencyAmount<Currency>;
     tokenInBefore: CurrencyAmount<Currency>;
     tokenOutAfter: CurrencyAmount<Currency>;
     tokenOutBefore: CurrencyAmount<Currency>;
+    tokenOutPortionRecipientBefore: CurrencyAmount<Currency>;
+    tokenOutPortionRecipientAfter: CurrencyAmount<Currency>;
```

Comment:
Don't think we should be returning 0 for the balance of the portion recipient if `portion` is not defined, seems like odd behavior. Instead we should be returning undefined or omitting the attributes entirely from the returned object. That way you don't have to check the definedness of `portionRecipientSigner` too